troubleshooting.md

```diff
@@ -0,0 +1,28 @@
+# Troubleshooting Android Client
+
+#### Can't Login?
+
+There are a few things you would need to first take note in logging in. Make sure that you enter a valid server for the URL address. Moreover, the username and password does not correspond to your OpenMRS ID. Follow the information on the link [here](https://openmrs.github.io/openmrs-android-client-user-guide/introduction.html) to log into the test server. 
+
+#### Have Another Issue?
+
+This guide covers the basic development setup and things to take note off while using the android client. Make sure the guide first. The guide walks through the basics of entering patient details, uploading patient photos, etc, which you could check out [here](https://openmrs.github.io/openmrs-android-client-user-guide/features.html).
```

Comment:
```suggestion
This guide covers the basic setup and instructions to take note of while using the Android client. Make sure to refer to the guide first. The guide walks through the basics of entering patient details, uploading patient photos, etc. which you may check out [in this page](https://openmrs.github.io/openmrs-android-client-user-guide/features.html).
```

Reply:
Note that this is a guide for the users / implementers, not strictly for new developers (although you can say that some instructions are for developers).